src/ExternalMenuTreeBuilder.php

```diff
@@ -183,7 +183,11 @@ private function createLink(
    *   Returns true or false.
    */
   private function inActiveTrail(object $item): bool {
-    if ($item->url->isRouted() && $item->url->getRouteName() === '<nolink>') {
+    if (
+      $item->url->isRouted() &&
+      $item->url->getRouteName() === '<nolink>' ||
+      $item->external
```

Comment:
This works on all other subdomains except avustukset.hel.fi, since it is not marked external.

See: [`helfi_api_base.services.yml`](https://github.com/City-of-Helsinki/drupal-module-helfi-api-base/blob/main/helfi_api_base.services.yml#L9)